From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 2 Aug 2023 12:14:56 -0700
Subject: [PATCH] test/test-sizeof: Include sys/timex.h for struct timex

Fixes

../git/src/test/test-sizeof.c:64:41: error: incomplete definition of type 'struct timex'
   64 |         check(typeof(((struct timex *)0)->freq), SIZEOF_TIMEX_MEMBER);
      |                      ~~~~~~~~~~~~~~~~~~~^

Upstream-Status: Backport [https://github.com/systemd/systemd/pull/28651]
---
 src/test/test-sizeof.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/test/test-sizeof.c b/src/test/test-sizeof.c
index 9d969cf8f1..b65c0bd370 100644
--- a/src/test/test-sizeof.c
+++ b/src/test/test-sizeof.c
@@ -4,6 +4,7 @@
 #include <string.h>
 #include <sys/resource.h>
 #include <sys/socket.h>
+#include <sys/timex.h>
 #include <sys/types.h>
 
 #define __STDC_WANT_IEC_60559_TYPES_EXT__
